src/methods/geom_relations/within.jl

```diff
@@ -82,6 +82,14 @@ true
 """
 within(g1, g2) = _within(trait(g1), g1, trait(g2), g2)
 
+"""
+    within(g1)
+
+Return a function that checks if a geometry is within `g1`.
```

Comment:
This is the opposite of the tests?

Reply:
Yeah I think I just wrote the docstring wrong.

Reply:
Thanks for catching that, should be all fixed now.  The perils of midnight coding...